src/components/People.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from "react";
+
+
+
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state = {
+            people: [],
+            userInput: ""
+        };
```

Comment:
```suggestion
        this.state = {
            people: [],
            userInput: "",
            selectedCharacter: ""
        };
```
You need another piece of state that keeps track of which character has been selected when your form is submitted.